alembic/versions/20231101_2d72d6876c52_remove_collection_external_integration.py

```diff
@@ -0,0 +1,273 @@
+"""Remove collection external integration.
+
+Revision ID: 2d72d6876c52
+Revises: cc084e35e037
+Create Date: 2023-11-01 22:42:06.754873+00:00
+
+"""
+import logging
+
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+from alembic import op
+from api.integration.registry.license_providers import LicenseProvidersRegistry
```

Comment:
Hmmm this is sticky isn't it... I don't think this will directly import the ORM models, since we don't do most of the imports until we create a registry, but creating a registry will import the models for sure. I'm not sure how having the models imported late like that will impact things... I'll make a couple tests.

Reply:
We imported the `LicenseProvidersRegistry` in a previous migration here:
https://github.com/ThePalaceProject/circulation/blob/4763d83386dde174a0a4ea7e9fa0cc66a394c7ae/alembic/versions/20230531_0af587ff8595_migrate_license_integrations_to_.py#L12

I think we've added columns since then, and it doesn't seem like its caused issues with the migration. I might be misunderstanding how this would break though, does that make sense?

Reply:
Yes, I tested this out by adding a new column in the Collections model, it did not fail due to a schema mismatch since we're not actively using the ORM (the models do get imported though).